tox.ini

```diff
@@ -88,17 +88,19 @@ deps =
     -r{toxinidir}/requirements.txt
 commands =
     coverage run --source={[vars]src_path} \
-        -m pytest --ignore={[vars]tst_path}unit -v {posargs}
+        -m pytest --ignore={[vars]tst_path}integration -v {posargs}
```

Comment:
This seems incorrect, integration test running when the env is for unit tests.

Reply:
this is the --ignore

Reply:
when you want to run integration tests, you ignore the unit tests, and vice versa. I fell for it, too